### PR TITLE
Fixes #13071 - add inline help for job_name placeholder

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: app/views/job_templates/_custom_tabs.html.erb [hunk 1]
```diff
@@ -10,7 +10,8 @@
 
   <%= text_f f, :description_format,
              :help_inline => popover(_('Explanation'), _('Description template determines the job name once it is submitted. Input values can become part of the name
-                         if they are specified using interpolation syntax, e.g. %{fqdn} where fqdn is the name of interpolated input.')) %>
+                         if they are specified using interpolation syntax, e.g. %{fqdn} where fqdn is the name of interpolated input.
+                         An exception is %{job_name} which is always replaced by template name.')) %>
```
Comment (stbenjam) [inline app/views/job_templates/_custom_tabs.html.erb]: %job_name renders as the *template* name? That sounds more like a bug.

Comment (ares) [inline app/views/job_templates/_custom_tabs.html.erb]: oops, it's a invocation job name, I took this from another PR that was replacing job name with template name, will fix the help

**Comment by ares** [comment]
@stbenjam I reworded it, does it make more sense now?

**Comment by stbenjam** [comment]
Not really :grin: The text itself is a little confusing to me. E.g. "Description template determines the job name." It doesn't, it determines the description.  

How about this?

*This template is used to generate the description. Input values can be used using the syntax %{fqdn}. You may also include the job name using %{job_name}.*

**Comment by adamruzicka** [comment]
+1 for @stbenjam's suggestion

**Comment by ares** [comment]
ok, I've changed it to your suggestion, just note that we display this description under job name column in job invocations table...

**Comment by ares** [comment]
[test]
```Fatal server error:
(EE) Server is already active for display 3
	If this server is no longer running, remove /tmp/.X3-lock
	and start again.
```

**Comment by stbenjam** [comment]
> ok, I've changed it to your suggestion, just note that we display this description under job name column in job invocations table...

Which also needs fixing at some point, it's not the job name

**Comment by ares** [comment]
agreed, got it captured from today meeting, I guess it's not blocking this though?

**Comment by stbenjam** [comment]
Nope, I didn't notice the problem was deeper than just that popover so we can fix the rest as part of a separate issue.  

ACK

**Comment by stbenjam** [comment]
Merging since I want the same description text in the API for https://github.com/theforeman/foreman_remote_execution/pull/103/

**Comment by stbenjam** [comment]
Thanks @ares

**Pull request merged**